test/gpu/native/studies/sort/radixSortGpu.chpl

```diff
@@ -433,33 +48,34 @@ module RadixSort {
     var prefixSum: [0..<buckets] uint;
 
     // Ceiling on number of iterations based on max element in array
+    var maxVal =  max(uint);
     // Reduce can be used to do this faster
+    // We do this because the preferred ways below either don't work
+    // or take too long
     // But I'm leaving this as is due to https://github.com/chapel-lang/chapel/issues/22736
-    var max = inputArr[0];
-    for i in 1..<inputArr.size {
-      if inputArr[i] > max {
-        max = inputArr[i];
-      }
-    }
+    // var maxVal = max reduce inputArr; // Doesn't work actually
+    // var maxVal = inputArr[0]; // Takes toooo long
+    // for i in 1..<inputArr.size {
+    //   if inputArr[i] > maxVal {
+    //     maxVal = inputArr[i];
+    //   }
+    // }
 
     // Number of iterations is the number of bits in the max element divided by number of bits we sort at a time
     var exp = 0;
-    while (max> 0){
-      if(noisy) then
-        writeln("       Exp: ", exp);
+    while (maxVal> 0){
+      if(noisy) then writeln("       Exp: ", exp);
```

Comment:
It is easy and unfair to pick on stylistic issues, but please try to be consistent within files. You have:  `if(noisy)` here, `if noisy` just below, `if (noisy)` just above.

I also realize that this is not introduced in this PR, but this also looks like a good PR to lump in some cosmetic improvements to the code.

Reply:
Just to rationalize a bit more: we point users at GPU tests more than non-GPU cases. So, it'd be good to make them look consistent in terms of style, ideally adhering also to the [Standard Module Style Guide](https://github.com/chapel-lang/chapel/blob/main/doc/rst/developer/bestPractices/StandardModuleStyle.rst) (it is about module style, but we hope to evolve it into a general set of style rules)